package.json

```
@@ -27,8 +27,8 @@
     "build:license": "tsx tools/license.ts",
     "build:logo": "tsx tools/startup-logo.ts",
     "test": "yarn test:types && yarn test:unit && yarn test:badge",
-    "test:cjs": "yarn --cwd tests/cjs && vitest run -r tests/cjs",
-    "test:esm": "yarn --cwd tests/esm && vitest run -r tests/esm",
+    "test:cjs": "yarn --cwd tests/cjs --ignore-engines && vitest run -r tests/cjs",
+    "test:esm": "yarn --cwd tests/esm --ignore-engines && vitest run -r tests/esm",
```

```suggestion
    "test:cjs": "yarn --cwd tests/cjs && vitest run -r tests/cjs",
    "test:esm": "yarn --cwd tests/esm && vitest run -r tests/esm",
```

revert when released